src/index.ts

```diff
@@ -289,7 +295,7 @@ export async function build(opts: BuildOptions) {
 	await runUserScripts(entryPath);
 	const extraFiles = await gatherExtraFiles(config.includeFiles, entryPath);
 
-	if (path.extname(entrypoint) === ".toml") {
```

Comment:
Can you briefly explain this? I haven't had much time to look over the new `runtime` config, but are they moving away from multiple lambdas?

Reply:
@mike-engel Indeed! In order to prevent people from using Runtimes to server render their frontend (which drastically decreases performance), Runtimes will – from now on - only be able to execute serverless functions (located in the `api` directory).

As you can see, the check still remains, but it's placed behind a version check. That way, we retain backwards compatibility but make the feature unavailable for when you're using zero config serverless functions.